react/src/components/FieldDisplayEditable.tsx

```diff
@@ -131,10 +136,6 @@ function EnhancedTextField({
             // Value is typed in, and can be really long (notes)
             textFieldProps.multiline = true;
             textFieldProps.maxRows = 2;
-        } else if (field.fieldName === "month_of_birth") {
-            textFieldProps.InputLabelProps = { shrink: true };
-            textFieldProps.type = "month";
-            textFieldProps.value = formatDisplayValue(field);
```

Comment:
Instead of losing the month picker in Chromium, if you lose the `formatDisplayValue(field)` here (e.g. just `field.value`) then the month field works in Firefox

Reply:
Yes, that fixes the problem. Right now, `birth_of_month` is a text input field in Firefox and it is a usable field.